### PR TITLE
Fixed spelling, casing, indentation, and removed unused variables for Remove-Student_Attributes.ps1

### PR DESCRIPTION
**Pull request by AyronJohnson**

### CODE REVIEW

File: SDS Scripts/Remove-Student_Attributes.ps1 [hunk 1]
```diff
@@ -1,56 +1,45 @@
...
-Set-AzureADUserExtension -ObjectId 73eb1ff2-0bb6-4d2f-9944-3414b1906869 -ExtensionName extension_fe2174665583431c953114ff7268b7b3_Education_EnglishLanguageLearnersStatus -ExtensionValue $remove
-}
+    $Ext = $User | Select-Object -ExpandProperty ExtensionProperty
+
+    #Determine if Student
+    if ($Ext.extension_fe2174665583431c953114ff7268b7b3_Education_ObjectType -like "Student") {
+
+        #Let the Admin know Student was found for processing
+        write-host -ForegroundColor green "Removing Sensitive Attributes for $DN"
+        $remove = " "
```
Comment (DanBaumgartner) [inline SDS Scripts/Remove-Student_Attributes.ps1:26]: Setting the extension value to " " doesn't remove the attribute container, just the value. Shouldn't we be using remove-azureaduserextension to completely remove the attribute?
https://docs.microsoft.com/en-us/powershell/module/azuread/remove-azureaduserextension?view=azureadps-2.0

File: SDS Scripts/Remove-Student_Attributes.ps1 [hunk 2]
```diff
@@ -1,56 +1,45 @@
...
+
+        #Remove each of the deprecated attributes 
+        Set-AzureADUserExtension -ObjectId 73eb1ff2-0bb6-4d2f-9944-3414b1906869 -ExtensionName extension_fe2174665583431c953114ff7268b7b3_Education_Gender -ExtensionValue $remove
```
Comment (DanBaumgartner) [inline SDS Scripts/Remove-Student_Attributes.ps1:29]: variable needs to be added for the "-ObjectId" parameter, not a hardcoded value

File: SDS Scripts/Remove-Student_Attributes.ps1 (hunk 1 above)
Comment (AyronJohnson) [reply SDS Scripts/Remove-Student_Attributes.ps1]: Done

File: SDS Scripts/Remove-Student_Attributes.ps1 (hunk 2 above)
Comment (AyronJohnson) [reply SDS Scripts/Remove-Student_Attributes.ps1]: Done

**Review by DanBaumgartner — APPROVED** [review]

**Pull request merged**